edb/server/protocol/auth_ext/data.py

```diff
@@ -112,12 +112,14 @@ class OAuthAccessTokenResponse:
     access_token: str
     token_type: str
     expires_in: int
-    refresh_token: str
+    refresh_token: str | None
 
     def __init__(self, **kwargs):
         for field in dataclasses.fields(self):
             if field.name in kwargs:
                 setattr(self, field.name, kwargs.pop(field.name))
+            else:
```

Comment:
FWIW I don't think you're supposed to overload the `__init__` method on dataclasses

Reply:
Never mind, it seems to be supported by the API

> init: If true (the default), a [__init__()](https://docs.python.org/3/reference/datamodel.html#object.__init__) method will be generated.

> If the class already defines [__init__()](https://docs.python.org/3/reference/datamodel.html#object.__init__), this parameter is ignored.